src/index.js

```diff
@@ -1,5 +1,8 @@
 const app = require('./app');
 const logger = require('./logger');
+const registerProcessEvents = require('./utils/processEvents');
+
+registerProcessEvents(logger);
```

Comment:
I do not like it at index level, this should move to the app.js

Reply:
i would keep it there, as this is not part of the application logic but should cover the whole app and be up before starting the app itself.